lib/rules/no-nested-splattributes.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const Rule = require('./base');
+
+const ERROR_MESSAGE = 'Nested splattributes are not allowed';
+
+module.exports = class NoNestedSplattributes extends Rule {
+  visitor() {
+    let splattributesParent = null;
+
+    return {
+      ElementNode: {
+        enter(node) {
+          let splattribute = node.attributes.find((it) => it.name === '...attributes');
+          if (splattribute) {
+            if (splattributesParent) {
+              this.log({
+                message: ERROR_MESSAGE,
+                line: splattribute.loc && splattribute.loc.start.line,
+                column: splattribute.loc && splattribute.loc.start.column,
+                source: this.sourceForNode(splattribute),
+              });
+            } else {
+              splattributesParent = node;
+            }
+          }
+        },
+        exit(node) {
+          if (splattributesParent === node) {
+            splattributesParent = null;
+          }
+        },
```

Comment:
Thoughts on using `path.parents` instead of manual book keeping + enter/exit? I'm sure you thought of it, just curious what led you to choosing this form instead (are we missing some API niceties or something)?

Reply:
yeah, would probably work too, but it felt a bit more efficient this way and not much more verbose than the alternative